js/config.js

```diff
@@ -258,14 +256,18 @@ function updateUserConfig(oldUserConfig) {
         userConfigFile = path.join(app.getPath('userData'), configFileName);
 
         if (!userConfigFile) {
-            return reject(`user config file doesn't exist`);
+            reject(`user config file doesn't exist`);
+            return;
         }
 
         // write the new user config changes to the user config file
-        fs.writeFileSync(userConfigFile, newUserConfigString, 'utf-8');
-
-        return resolve();
-
+        fs.writeFile(userConfigFile, newUserConfigString, 'utf-8', (err) => {
+            if (err) {
+                reject(`Failed to update user config error: ${err}`);
+                return;
```

Comment:
this can be moved to the above line?

Reply:
Then we would also need to `return fs.writeFile` which I think is not a good idea